src/tests/test_stashcache.py

```diff
@@ -50,14 +50,14 @@ def test_allowedVO_includes_ANY_for_ligo_inclusion(self, client: flask.Flask, mo
 
         stashcache.generate_cache_authfile(global_data, "osg-sunnyvale-stashcache.t2.ucsd.edu")
 
-        assert spy.call_count == 1
+        assert spy.call_count == 4
```

Comment:
Increase call count: a new LIGO namespace got added
```suggestion
        assert spy.call_count == 5
```